src/ui/Base/QueryBuilder.ts

```diff
@@ -354,8 +356,8 @@ export class QueryBuilder {
       sortField: this.sortField,
       queryFunctions: this.queryFunctions,
       rankingFunctions: this.rankingFunctions,
-      groupBy: this.validateGroupBy(),
-      facets: this.validateFacets(),
+      groupBy: this.groupBy,
```

Comment:
So, right now, I'm thinking that this would fails in the Search API if both `groupBy` and `facets` if it's a bad request, which is okay.
And could potentially become a "correct" request in the future, which is also the goal of this change, if I remember correctly.

Now, we should think about handling that properly in the error message component.

Can we do anything to help display a meaningful error message when a bad request come back from the API for this error ?
So that we don't get maintenance case request when those errors happens ?

Reply:
The error returned by the SearchAPI looks descriptive enough in the browser console. Should we make our own on top of it?
![Screen Shot 2019-03-28 at 7 05 45 PM](https://user-images.githubusercontent.com/4923043/55198482-84fdaf00-518c-11e9-9a95-cddbdd9cec34.png)